V5 Release note - Governance part (#877)
akeneo/pim-helpcenter: content/updates/governance/3-Revamped-yes-no-attribute.md

# Meet the new Yes/No attribute

## New Yes/No attribute in the product form
::: meta-data type="Improvement" ee-only="false"

> The Yes/No attribute is now more user-friendly: you can easily see if your Yes/No attribute has no value yet. You can also empty it if needed.

Reviewer: ```suggestion
The Yes/No attribute is now more user-friendly: you can easily see if a value has been set or not yet. You can also empty it if needed.
```

![New Yes/No attribute design](../img/new-yes-no-attribute-design.png)

## Better filter with the Yes/No attribute
::: meta-data type="New" ee-only="false"

> Two new operators `is empty` and `is not empty` are now available in the product grid and in the [Rules Builder](TODO), to help you filter on Yes/No attributes. These new filters will help you to easily find what you were looking for in the product grid and be even more precise while building your rules.

Reviewer: ```suggestion
Two new operators `is empty` and `is not empty` are now available in the product grid and in the [Rules Builder](TODO), to help you filter on Yes/No attributes. These new filters will help you to easily find what you are looking for in the product grid and be even more precise while building your rules.
```

> Two new operators `is empty` and `is not empty` are now available to help you filter on Yes/No attributes in the product grid and to automate enrichment with the rules engine. These new filters help you boost your productivity and be even more efficient when looking for specific products and product models.

Reviewer 2: ```suggestion
Two new operators `is empty` and `is not empty` are now available in the product grid and in the Rules Builder, to help you filter on Yes/No attributes. These new filters will help you to easily find what you were looking for in the product grid and be even more precise while building your rules.
```
Can we put a link on Rules Builder to link it to the relevant section in Automation?

Author: ```suggestion
Two new operators `is empty` and `is not empty` are now available in the product grid and in the [Rules Builder](TODO), to help you filter on Yes/No attributes. These new filters will help you to easily find what you were looking for in the product grid and be even more precise while building your rules.
```

> Two new operators `is empty` and `is not empty` are now available in the product grid and in the [Rules Builder](../articles/manage-your-rules.html#the-rule-builder-tab), to help you filter on Yes/No attributes. These new filters will help you to easily find what you were looking for in the product grid and be even more precise while building your rules.

Author: ```suggestion
Two new operators `is empty` and `is not empty` are now available in the product grid and in the [Rules Builder](../articles/manage-your-rules.html#the-rule-builder-tab), to help you filter on Yes/No attributes. These new filters will help you to easily find what you are looking for in the product grid and be even more precise while building your rules.
```

![New is empty and is not empty filters on the Yes/No attributes](../img/is-empty-is-not-empty-filters.png)

The first one, `is empty`, allows you to find products and product models that have no value for a specific Yes/No attribute. The second one, `is not empty`, allows you to search for the opposite.

## Default value for the Yes/No attribute
::: meta-data type="New" ee-only="false"

> You can now define a default value for each of your Yes/No attributes. This will definitely help you be way more productive and swiftly reach your completeness goals. This new feature can also be used to initialize workflows based on Yes/No attributes.

Reviewer: ```suggestion
You can now define a default value for each of your Yes/No attributes. This will definitely help you be more productive and swiftly reach your completeness goals. This new feature can also be used to initialize workflows based on Yes/No attributes.
```

> You can now define a default value for each of your Yes/No attributes. This will definitely help you  be way more productive and swiftly reach your completeness goals. This new feature can also be used to initialize workflows based on Yes/No attributes.

Reviewer 3: ```suggestion
You can now define a default value for each of your Yes/No attributes. This will definitely help you be way more productive and swiftly reach your completeness goals. This new feature can also be used to initialize workflows based on Yes/No attributes.
```
